release/release_tests.yaml

```
@@ -1974,6 +1974,54 @@
 
   alert: default
 
+- name: deployment_graph_long_chain
+  group: Serve tests
+  working_dir: serve_tests
+
+  legacy:
+    test_name: deployment_graph_long_chain
+    test_suite: serve_tests
+
+  frequency: nightly
+  team: serve
+
+  cluster:
+    cluster_env: app_config.yaml
+    cluster_compute: compute_tpl_single_node_32_cpu.yaml
+
+  run:
+    timeout: 3600
+    long_running: false
+    script: python workloads/deployment_graph_long_chain.py --chain-length=10 --num-clients=4 --local-test=False
+    type: sdk_command
+    file_manager: job
+
+  alert: default
```

Can we mark these as unstable until we observed them passing over say 2 weeks?

```suggestion
  alert: default
  stable: False
```